pipenv/core.py

```diff
@@ -126,16 +126,15 @@ def do_clear():
 
     try:
         vistir.path.rmtree(PIPENV_CACHE_DIR, onerror=vistir.path.handle_remove_readonly)
-        vistir.path.rmtree(
-            locations.USER_CACHE_DIR, onerror=vistir.path.handle_remove_readonly
-        )
     except OSError as e:
         # Ignore FileNotFoundError. This is needed for Python 2.7.
         import errno
 
         if e.errno == errno.ENOENT:
             pass
         raise
+    # Other processes may be writing into this directory simultaneously.
+    vistir.path.rmtree(locations.USER_CACHE_DIR, ignore_errors=True)
```

Comment:
yeah i guess that's fine, or maybe `ignore_errors=environments.PIPENV_IS_CI`

Reply:
(but I'm not sure that's necessary, and don't delay merging over it)

Reply:
I didn't know that env var, that looks better.